jdbc-interface/src/main/java/ch/unibas/dmi/dbis/polyphenydb/jdbc/DbmsMeta.java

```diff
@@ -981,7 +988,9 @@ public ExecuteResult prepareAndExecute( final StatementHandle h, final String sq
                         h.id,
                         false,
                         signature,
-                        maxRowsInFirstFrame > 0 ? fetch( h, 0, (int) Math.min( Math.max( maxRowCount, maxRowsInFirstFrame ), Integer.MAX_VALUE ) ) : Frame.MORE // Send first frame to together with the response to save a fetch call
+                        maxRowsInFirstFrame > 0 && SEND_FIRST_FRAME_WITH_RESPONSE
+                                ? fetch( h, 0, (int) Math.min( Math.max( maxRowCount, maxRowsInFirstFrame ), Integer.MAX_VALUE ) )
+                                : null //Frame.MORE // Send first frame to together with the response to save a fetch call
```

Comment:
In the inline else case, this now returns `null` instead of `Frame.MORE`. Is this intended behaviour?
Additionally it seems that this if will always evaluate to false, and execute the else branch.

Reply:
Yes, this is intended and necessary due to a bug in Avatica.

Reply:
Why do you think it always evaluated to false?

Reply:
`SEND_FIRST_FRAME_WITH_RESPONSE` is always false, so we get:

`maxRowsInFirstFrame > 0 && SEND_FIRST_FRAME_WITH_RESPONSE` -> `maxRowsInFirstFrame > 0 && false` -> `false`

Unless `SEND_FIRST_FRAME_WITH_RESPONSE` is modified elsewhere, which would be a bad design.

Reply:
Sorry, I misunderstood your comment. Yes, this is correct and intended. Due to a bug in avatica it fails under some circumstances when sending data together with the first frame. So I "disabled" that, but left the code to have a closer look on that in future.

Reply:
Then I think this behaviour should be documented with the code, possibly including a `fixme` or `todo` comment.

Reply:
Yes, this is right. I will add a todo comment